tmail-backend/apps/memory/src/test/java/com/linagora/tmail/james/app/MemoryServerWithoutOpenPaasTest.java

```diff
@@ -0,0 +1,34 @@
+package com.linagora.tmail.james.app;
+
+import static org.apache.james.data.UsersRepositoryModuleChooser.Implementation.DEFAULT;
+
+import org.apache.james.JamesServerBuilder;
+import org.apache.james.JamesServerExtension;
+import org.apache.james.utils.GuiceProbe;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.google.inject.multibindings.Multibinder;
+import com.linagora.tmail.encrypted.MailboxConfiguration;
+import com.linagora.tmail.encrypted.MailboxManagerClassProbe;
+import com.linagora.tmail.module.LinagoraTestJMAPServerModule;
+
+public class MemoryServerWithoutOpenPaasTest {
```

Comment:
I think all the other tests check what happens without the openpaas module enabled.

As it stands this test is useless.

We shall rather check the exception we get when we try to start tmail with the openpaas extension without configuring RabbitMQ...